dashboards/dojo/dojo.json

```diff
@@ -0,0 +1,1154 @@
+{
```

Comment:
This general feedback applies to all of our quickstarts. When we have only one dashboard, it feels odd to use the number:

"Dojo quickstart contains 1 dashboard."

I'd prefer it if it said something like:

"Dojo quickstart contains one dashboard."

In general, our style guide recommends using words for numbers 0-9, and this is generally considered best practice.

Another solution would be to simply delete this first sentence across the board, everywhere. The header includes a very prominent number "Dashboards 1" and this sentence doesn't add much.

Feel free to reach out to me for more info. I realize this section of the quickstarts is generated programmatically and may not be trivial to update.

Reply:
@paperclypse correct, this is part of the template.  In 3 weeks we are kicking off a project to migrate Public I/O to a different content management system, and we'll use that subsequent opportunity to rework the design, layout and messaging across the board